### ISSUE TITLE
Profiles not found, when Windows Terminal is installed e.g. via scoop

### ISSUE THREAD

**Issue by guntern** [paradox00/Flow.Launcher.Plugin.WindowsTerminal#5]
Hi,
I noticed that the profiles are not found on a newly installed machine. After some digging I figured out, that this might be related to installing Windows Terminal via e.g. scoop. We do not have access to the Windows Store and thus I had to use alternative installation methods. Using scoop seemed to be the most reasonable to ease the update process.

Currently the plugin expects the settings under:
`<User>\AppData\Local\packages\Microsoft.WindowsTerminal_...\LocalState\settings.json`

When Windows Terminal is installed via scoop, the settings are located under:
`<User>\AppData\Local\Microsoft\Windows Terminal\settings.json`

Furthermore the starting of Windows Terminal seems to be different. Currently the plugin uses a way to start the app via `ctypes.windll.shell32.ShellExecuteW`. This does not work for the scoop version.

I figured out a quick way to solve the settings issue, but did not figure yet a way out to solve the starting issue.

**Comment by guntern** [comment]
I fiddled around with other ways to start the actual terminal. Is there a specific reason you (@paradox00) use `ctypes.windll.shell32.ShellExecuteW` instead of e.g. `subprocess.run`?

**Comment by paradox00** [comment]
Yes, `subprocess.run` doesn't support invocation of windows apps using the shell extension. Executing the `wt.exe` doesn't let you choose which version to run (beta or release).

**Comment by guntern** [comment]
> Yes, `subprocess.run` doesn't support invocation of windows apps using the shell extension. Executing the `wt.exe` doesn't let you choose which version to run (beta or release).

I see. I thought I was on the right track, but apparently not.

It may get tricky to determine the version of Windows Terminal to start, depending on the found settings file.

**Comment by paradox00** [comment]
You can always add new configuration option to choose to invoke `wt.exe` instead of executing the package directly...

**Comment by guntern** [comment]
A user option how to invoke `wt.exe` sounds great. One could also add a configuration to specify the path to the `settings.json`, so that the plugin would not have to test it. 
But I actually like your idea of supporting multiple installations of windows terminal. Let's see, when I have time for this again.
